### PR TITLE
Tomasmaster2

### PR DESCRIPTION
**Pull request by robinschmid**
See PR #539 

Changed title of show MS/MS and added tooltip.

### CODE REVIEW

File: src/main/java/net/sf/mzmine/modules/visualization/peaklisttable/PeakListTablePopupMenu.java [hunk 1]
```diff
@@ -33,7 +33,6 @@
 import javax.swing.ListSelectionModel;
 import javax.swing.SwingUtilities;
 import javax.swing.table.AbstractTableModel;
-import com.google.common.collect.Range;
```
Comment (tomas-pluskal) [inline src/main/java/net/sf/mzmine/modules/visualization/peaklisttable/PeakListTablePopupMenu.java]: Removing this import causes compilation to fail. Why did you remove it?

Comment (robinschmid) [reply src/main/java/net/sf/mzmine/modules/visualization/peaklisttable/PeakListTablePopupMenu.java]: Thats weird. I did not intentionally delete it. Maybe the auto import module in eclipse accidentally deleted it.
Sorry I thought I only did two small changes.
Can you change and reimport it?
I could do it tomorrow.

Thanks for your review. And sorry for that problem.

**Comment by tomas-pluskal** [comment]
No problem, I'll fix that.
